Gemfile.lock

```diff
@@ -1085,4 +1151,4 @@ RUBY VERSION
    ruby 2.6.1p33
 
 BUNDLED WITH
-   1.17.3
+   2.0.1
```

Comment:
I'm not sure if you intentionally updated bundler along with some dependencies but I'm having trouble running the app locally now (something with `push-notifications-ruby` is messing me up). Heroku also does not support Bundler 2.0 without additional configuration. Can you please revert these changes.

Reply:
Oops, that was not intentional. Probably I messed this up while rebasing. Will change back.